xml/System.Diagnostics/Activity.xml

```diff
@@ -375,9 +377,9 @@
         <Parameter Name="parentId" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="parentId">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="parentId">The id of the parent operation.</param>
+        <summary>Updates this <see cref="T:System.Diagnostics.Activity" /> to indicate that the <see cref="T:System.Diagnostics.Activity" /> with ID <paramref name="parentId" /> caused this <see cref="T:System.Diagnostics.Activity" />. This is intended to be used only at 'boundary' scenarios where an <see cref="T:System.Diagnostics.Activity" /> from another process logically started this <see cref="T:System.Diagnostics.Activity" />. The <paramref name="parentId" /> brings up the <see cref="P:System.Diagnostics.Activity.Tags" /> (as well as the <see cref="P:System.Diagnostics.Activity.ParentId" /> property), and can be used to reconstruct the causal tree.</summary>
```

Comment:
Should be `in boundary scenarios` rather than `at boundary scenarios`

Reply:
```suggestion
        <summary>Updates this <see cref="T:System.Diagnostics.Activity" /> to indicate that the <see cref="T:System.Diagnostics.Activity" /> with ID <paramref name="parentId" /> caused this <see cref="T:System.Diagnostics.Activity" />. This is intended to be used only in 'boundary' scenarios where an <see cref="T:System.Diagnostics.Activity" /> from another process logically started this <see cref="T:System.Diagnostics.Activity" />. The <paramref name="parentId" /> brings up the <see cref="P:System.Diagnostics.Activity.Tags" /> (as well as the <see cref="P:System.Diagnostics.Activity.ParentId" /> property), and can be used to reconstruct the causal tree.</summary>
```